feature/experimental/p4rt/tests/p4rt_election/p4rt_election_test.go

```diff
@@ -316,21 +316,27 @@ func TestZeroMaster(t *testing.T) {
 		highID:     inId0,
 		handle:     clientConnection(t, dut),
 		deviceID:   deviceId,
-		wantFail:   true,
-		wantStatus: 3,
+		wantFail:   false,
+		wantWrite:  false,
+		wantRead:   true,
+		wantStatus: 0,
 	}
 	t.Run(test.desc, func(t *testing.T) {
 		resp, err := streamP4RTArb(&test)
-		if err == nil && test.wantFail {
-			t.Errorf("Zero ElectionID (0,0) is allowed: %v", err)
+		if err != nil && test.wantFail {
 			removeClient(test.handle)
-		} else {
-			t.Logf("Zero ElectionID (0,0) connection failed as expected: %v", err)
+			t.Errorf("Zero ElectionID (0,0) is rejected by P4RT server: %v", err)
+			return
 		}
+		t.Logf("Zero ElectionID (0,0) connection success as expected: %v", err)
```

Comment:
Also this is not required here. There won't be any `err` when this line is reached.

Reply:
Removed wantFail and err